tests/tests_api_v1_1/test_filters.py

```diff
@@ -1,15 +1,19 @@
 # coding: utf-8
```

Comment:
Please can you apply only changes due to pendulum upgrade. If this file requires other changes, submit another PR

Reply:
removed